index.html

```diff
@@ -473,6 +473,21 @@ <h1>Download</h1>
 						<div class="col-md-6"><img src="images/sbc_images/parallels.svg" alt="Parallels logo" width="8" height="5" loading="lazy"></div>
 					</div>
 				</div>
+				<div id="utm" class="toggleDiv single-project">
+					<div class="row"><div class="col-md-12 project-title"><h3>UTM</h3><span class="show_hide close"></span></div></div>
+					<div class="row">
+						<div class="col-md-6 project-description">
+							<div class="project-info">
+								<div><span>RAM:</span>1 GiB | Can be changed</div>
+								<br>
+								<div><a href="https://dietpi.com/downloads/images/DietPi_UTM-x86_64-Bullseye.7z"><span>Download Image:</span><i class="fa fa-download"></i><u>Download</u></a></div>
+								<div><a href="https://dietpi.com/docs/install/" target="_blank" rel="noopener"><span>How to install / Details:</span><i class="fa fa-book"></i><u>View</u></a></div>
+							</div>
+							<p>The UTM virtual machine is great for those occasions where SBC performance just isn't enough. Run one of these on any x86_64 PC/server and still get the same great DietPi features and experience.</p>
```

Comment:
We could tail this generic VM text a bit for the virtualizer, e.g. UTM is nice because you can even run it on iOS phones and tablets. Means an `aarch64` would be nice, not only for M1 but also for native iPhone virtualization without emulation.